package.json

```diff
@@ -33,14 +33,19 @@
   "browser": "./lib/nodejs/lib/thrift/browser.js",
   "main": "./lib/nodejs/lib/thrift",
   "engines": {
-    "node": ">= 4.1.0"
```

Comment:
This is a massive version jump of ~4 years. Are we sure want this?

At the very least [LANGUAGES.md](https://github.com/apache/thrift/blob/master/LANGUAGES.md) should be updated to reflect this.

Reply:
I assume you are right, we may not require this new node version, its just the one we tested with. I'll try to clarify what is required.

Reply:
The version jump of ~4 years is huge, I understand but do you think it is a good idea to work with versions which are no longer maintained? 

If this is a necessary requirement I could switch to node version '6.17.1'  which was the last release of LTS version 'Boron' and has some security fixes in it: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V6.md#6.17.1

Reply:
I'd be happy to raise the minimum version to the latest supported LTS release.

Reply:
The latest supported LTS release would be 8.16.2: https://nodejs.org/en/about/releases/ (the end of life will be in 2 weeks 2019-12-31 😅) 

I just mentioned 6.17.1 because the version jump wouldn't be that big, but I still won't recommend to use that. If you think we should be careful with these version changes, we should go for that. I'll update [LANGUAGES.md](https://github.com/apache/thrift/blob/master/LANGUAGES.md) and packages.json

Reply:
OK then, let's update the min version to 10x which should be supported until mid-2021.